Vic2ToHoI4/Source/V2World/Localisations/Vic2Localisations.h

```diff
@@ -0,0 +1,50 @@
+#ifndef VIC2_LOCALISATIONS_H
+#define VIC2_LOCALISATIONS_H
+
+
+
+#include <map>
+#include <optional>
+#include <string>
+#include <unordered_map>
+
+
+
+namespace Vic2
+{
+
+using LanguageToLocalisationMap = std::map<std::string, std::string>;
+using KeyToLocalisationsMap = std::unordered_map<std::string, LanguageToLocalisationMap>;
```

Comment:
Why unordered? When filling it in the order is enforced and later all lookups will be faster.

Reply:
Ultimately because that's what was there before. But...

map lookup is O(log n)
unordered_map lookup is O(n) (it uses a hash map)

Though if we really cared about performance, we'd pull in Abseil and use their ones. No kidding 1000x faster in every scenario you can think of.